x-pack/plugin/core/src/main/resources/org/elasticsearch/xpack/profiler/ilm-policy/profiling.json

```diff
@@ -1,12 +1,10 @@
 {
   "phases": {
     "hot": {
-      "min_age": "0ms",
       "actions": {
         "rollover": {
           "max_primary_shard_size": "50gb",
-          "max_age": "7d",
-          "min_docs": 1
+          "max_age": "30d"
```

Comment:
With this policy change we avoid creating too many K/V indices (which will contain duplicates) yet event indices will be rolled over based on size if needed. `min_docs` is not needed as ILM does not rollover empty indices by default.